### ISSUE TITLE
swagger-node-express solved - new issue

### ISSUE THREAD

**Issue by JereMc** [swagger-api/swagger-node#291]
Working with MEAN stack.  I added swagger-node-express to the dependencies in which solved one issue, but then when I tried to run gulp I got this

Invoking gulp - development
[00:20:00] Using gulpfile /var/www/jammeantest/gulpfile.js
[00:20:00] Starting 'clean'...
[00:20:00] Finished 'clean' after 4.71 ms
[00:20:00] Starting 'default'...
[00:20:00] Starting 'coffee'...
[00:20:00] Finished 'coffee' after 6.27 ms
[00:20:00] Starting 'clean'...
[00:20:00] Starting 'less'...
[00:20:00] Starting 'csslint'...
[00:20:00] Starting 'env:development'...
[00:20:00] Finished 'env:development' after 20 μs
[00:20:00] Starting 'devServe'...
[00:20:00] Finished 'devServe' after 73 ms
[00:20:00] Starting 'watch'...
[00:20:01] Finished 'watch' after 1.35 s
[00:20:01] Finished 'clean' after 1.66 s
[00:20:01] [nodemon] v1.4.1
[00:20:01] [nodemon] to restart at any time, enter `rs`
[00:20:01] [nodemon] watching: *.*
[00:20:01] [nodemon] starting `node --debug server.js`
debugger listening on port 5858
[00:20:02] Finished 'less' after 1.93 s
[00:20:02] 
[00:20:02] csslint: 3 files lint free
[00:20:02] Finished 'csslint' after 2.02 s
[00:20:02] Starting 'development'...
[00:20:02] Finished 'development' after 9.7 μs
/var/www/jammeantest/packages/core/users/public/assets/lib/angular-jwt/dist/angular-jwt.min.js not found
/var/www/jammeantest/packages/core/admin/public/assets/lib/ng-clip/src/ngClip.js not found
/var/www/jammeantest/packages/core/admin/public/assets/lib/zeroclipboard/dist/ZeroClipboard.js not found
/var/www/jammeantest/packages/custom/i18n/public/assets/lib/i18next/i18next.js not found
/var/www/jammeantest/packages/custom/i18n/public/assets/lib/ng-i18next/dist/ng-i18next.js not found

/var/www/jammeantest/node_modules/meanio/node_modules/q/q.js:155
                throw e;
                      ^
TypeError: Object #<Object> has no method 'createNew'
    at /var/www/jammeantest/packages/core/swagger/app.js:14:45
    at Dependency.Dependable.runAction (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:73:22)
    at Dependency.runAction (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:98:48)
    at Dependency.Dependable.fire (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:70:53)
    at Dependency.fire (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:93:38)
    at Consumer.Dependable.tryResolve (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:67:24)
    at traverseArrayConditionally (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:20:11)
    at Consumer.Dependable.resolve (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:50:3)
    at Meanio.Container.resolve (/var/www/jammeantest/node_modules/meanio/node_modules/lazy-dependable/index.js:195:5)
    at moduleActivator (/var/www/jammeantest/node_modules/meanio/lib/core_modules/module/index.js:81:21)
    at ListItem.apply (/var/www/jammeantest/node_modules/meanio/node_modules/complex-list/index.js:22:12)
    at ModuleList.List.traverse (/var/www/jammeantest/node_modules/meanio/node_modules/complex-list/index.js:134:10)
    at enableModules (/var/www/jammeantest/node_modules/meanio/lib/core_modules/module/index.js:101:12)
    at findModulesDone (/var/www/jammeantest/node_modules/meanio/lib/core_modules/module/index.js:69:3)
    at _fulfilled (/var/www/jammeantest/node_modules/meanio/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/var/www/jammeantest/node_modules/meanio/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/var/www/jammeantest/node_modules/meanio/node_modules/q/q.js:796:13)
    at /var/www/jammeantest/node_modules/meanio/node_modules/q/q.js:604:44
    at runSingle (/var/www/jammeantest/node_modules/meanio/node_modules/q/q.js:137:13)
    at flush (/var/www/jammeantest/node_modules/meanio/node_modules/q/q.js:125:13)
    at process._tickCallback (node.js:415:13)
[00:20:04] [nodemon] app crashed - waiting for file changes before starting...

**Comment by iamike** [comment]
never get me pass.....? same as me, I am in madness mode.
![screen shot 2015-12-02 at 3 16 16 pm](https://cloud.githubusercontent.com/assets/526854/11524674/ac3d7f0a-9907-11e5-86c5-79b635c3b7d4.png)

**Comment by jclarost** [comment]
me too, any body know, how can it solve.
